cluster/addons/registry/registry-pv.yaml.in

```diff
@@ -1,10 +1,7 @@
-kind: PersistentVolume
 apiVersion: v1
```

Comment:
These files are inserted into https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/registry/README.md via some munging. You need to run the munger to add updates to that file into your PR.

Reply:
@roberthbailey Thank you, I've fixed it.